linkerd/examples/mutualTls.yaml

```diff
@@ -16,4 +25,4 @@ routers:
       # clients must provide a valid certificate
```

Comment:
Shouldn't we have a way of setting the intermediate certificate chain above for the server cert too, like we do for the client cert? In other words, `(cert, intermediates, key)` should always go together.